winres/winres.json

```diff
@@ -0,0 +1,28 @@
+{
+  "RT_VERSION": {
+    "#1": {
+      "0000": {
+        "fixed": {
+          "file_version": "0.0.0.1",
+          "product_version": ""
+        },
+        "info": {
+          "0409": {
+            "Comments": "",
+            "CompanyName": "",
+            "FileDescription": "",
+            "FileVersion": "",
+            "InternalName": "",
+            "LegalCopyright": "",
+            "LegalTrademarks": "",
+            "OriginalFilename": "",
+            "PrivateBuild": "",
+            "ProductName": "",
+            "ProductVersion": "",
+            "SpecialBuild": ""
```

Comment:
Do we have to set all these empty fields? If we omit them, will they get "" as their default value?
Might be worth setting `CompanyName` and `ProductName`?
Or make use of `go-winres simply` which is `a simpler make that does not rely on a json file.` ?

Reply:
> Do we have to set all these empty fields?

I mean, do we have to explicit list them in the json file if their value is ""? If we omit them, will they get set as "" as a default?